docs/guides/faq.html.md

```diff
@@ -0,0 +1,28 @@
+---
+title: FAQ
+subtitle: Plugin creation
+layout: guide
+order: 0
+---
+
+This ensures that Danger only runs when you have the environment variables set up to run.
+
+### Can Danger comment inline on an PR?
```

Comment:
I changed the wording, it should make more sense without knowing the terminology